paper-styles.html

```diff
@@ -30,6 +30,19 @@
 We recommend importing each of these individual files, and using the style mixins
 available in each ones, rather than the aggregated `paper-styles.html` as a whole.
 
+Create your own global theme to override components that use paper-styles by creating a component file as follows and importing it into your index.html.
+```
+<link rel="import" href="bower_components/polymer/lib/elements/custom-style.html">
+<custom-style>
+  <style is="custom-style">
+    html {
+      --primary-color: var(--paper-green-500);
+      --accent-color: var(--paper-deep-orange-500);
+    }
+  </style>
+</custom-style>
+```
+
```

Comment:
Would you mind adding a line about what "importing it in your index.html" actually means, since we're adding extra docs? :)

Reply:
Good point, that wording was wonky. How does it look now?